docs/_pages/docs/03-understand/09-swaps.md

```diff
@@ -8,6 +8,73 @@ sidebar:
 src: "_pages/docs/03-understand/09-swaps.md"
 ---
 
-<!-- TODO: explain and show (UI) how to do P2P LN<-> onchain swaps -->
+Apart from the various fiat payment methods available, there are something
+called *Swap Destinations* as well. These are payment methods, but for BTC, on a
+network other than the Lightning Network. This is usefull if you want to
+exchange lightning satoshis for on-chain satoshis (or on any other network like
+Liquid BTC if you prefer). This process of exchanging satoshis on the the
+lightning network for on-chain satoshis is usually referred to as a "swap".
 
-{% include wip %}
\ No newline at end of file
+The table below makes it simple to understand swap in terms of "buyer" and "seller":
+
+| Side   | Sends         | Recieves     | Swap type |
+|--------|---------------|--------------|-----------|
+| Seller | ⚡BTC         | 🔗 BTC       | Swap out  |
+| Buyer  | 🔗 BTC        | ⚡BTC        | Swap in   |
+
+
+### How to do a P2P Swap
+
+If you want to recieve satoshis over the Lignthing Network in exchange for your
+on-chain satoshis, then you create a **BUY** order. On the contrary, if you want to
+recieve satoshis on-chain in exchange for your Lignthing Network satoshis, then
+create a *SELL* order.
+
+In the Order Create screen, select "BTC" from the currency dropdown menu:
+
+<div align="center">
+    <img src="/assets/images/understand/btc-swap-in-dropdown.png"/>
+</div>
+
+Select your Swap Destination from the dropdown:
+
+<div align="center">
+    <img src="/assets/images/understand/swap-destination-selection.png"/>
+</div>
+
+You then set the amount or range you want to swap for. Remember that if you
+are a seller, then you will recieve on-chain BTC and if you are the buyer, you
+will be sending on-chain BTC:
```

Comment:
We'll eventually rework the UI for swaps and move it away from fiat currencies. It is certainly not intuitive at all at the moment

Reply:
Yup, but I also find it sort of elegant that it just "fits in" the rest of the payment method. But, certainly, a separate ui for swaps would be clearer. Also, since it's on-chain, we could add tons of other things, like adding tracking the address (by having a dedicated field for it) and check if the amount is correct and wait for confirmation on-chain, many possibilities there to make it streamlined.